woff1/tools/validator/validator.py

```diff
@@ -576,7 +577,7 @@ def _testHeaderSignature(data, reporter):
     - The signature must be "wOFF".
     """
     header = unpackHeader(data)
-    signature = header["signature"]
+    signature = header["signature"].decode()
```

Comment:
Binary data needs to be decoded in order to be able to perform string comparison in Python 3.